site/content/guides/claude-code-monitoring.md

```diff
@@ -0,0 +1,415 @@
+
+```{title="Prompt:"}
+# When writing a Playwright check, if you find multiple elements that match a locator, make sure the locator includes .first() to take the first result.
+```
+This solved the general issue for new browser checks created by Claude Code.
+
+### 4. Going Further: Edit Multiple Checks and Configurations
+
+Another use case that may be useful for larger projects, try prompting Claude Code with something like:
+
+```{title="Prompt:"}
+update every `.check.ts` file for a check that touches [danube-web.shop](http://danube-web.shop/), and change the frequency to every 30 minutes.
+```
+
+Not only did this request work, but even produced sensible feedback about the checks that currently had no frequency setting:
+
+```{title="Reply from Claude Code:"}
+  The url.check.ts and api.check.ts files don't have individual frequency settings -
+  they inherit the default frequency from the project configuration in
+  checkly.config.ts.
+```
+
+Once we're done creating new checks, it's time to run them all.
+
+
+## Deploy With the Checkly CLI
+
+In our example, we want to deploy a couple of checks at once, so it's worth checking whether the configuration is properly formatted before deployment. Many problems would have been revealed while running `test`, but if, for example, the [logical IDs](https://www.checklyhq.com/docs/cli/constructs-reference/#synthetic-checks) of current and existing checks are colliding, we'll need to run `deploy` to detect that. Run the `deploy` command with the `-p` preview flag:
+
+```bash
+npx checkly deploy -p
+```
+
+This command won't deploy anything, just parse our project and show us what will be created when we run the command without the preview flag. When you first run `deploy -p` you should just see a long list of created tests in the deployment command. For this tutorial, here's the preview of a more complex project state:
+
+```bash
+Bundling project resources... ✅
+
+Create:
+    BrowserCheck: __checks__/crime-category.spec.ts
+    BrowserCheck: __checks__/fantasy-category.spec.ts
+    BrowserCheck: __checks__/homepage-elements.spec.ts
+    BrowserCheck: __checks__/navigation-menu.spec.ts
+    BrowserCheck: __checks__/search-functionality.spec.ts
+    BrowserCheck: book-detail-visibility-check-v1
+    BrowserCheck: signup-form-elements-check-v1
+
+Delete:
+
+    Check: danube-web-shop-monitor
+
+Update and Unchanged:
+    BrowserCheck: __checks__/homepage.spec.ts
+```
+
+Let's discuss the output of this preview command in detail, as it's critical to understand how Checkly handles deployments from your project:
+* Checks that have no .check.ts config file are listed by their filename and will be named for their file name in the Checkly web UI. In general, it's better to include a config file for every check since it leaves you in explicit control of the check's logical ID. We've left out the config for five checks for demonstration purposes.
```

Comment:
```suggestion
* Checks that have no `.check.ts` config file are listed by their filename and will be named for their file name in the Checkly web UI. In general, it's better to include a config file for every check since it leaves you in explicit control of the check's logical ID. We've left out the config for five checks for demonstration purposes.
```